docs/Move-Item2.md

```diff
@@ -0,0 +1,139 @@
+---
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Move-Item2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Move an item from one location to another.  Locations are not limited by 256 character path length.